.gitmodules

```diff
@@ -33,7 +33,7 @@
 	shallow = true
 [submodule "musl"]
 	path = musl
-	url = https://git.musl-libc.org/git/musl
+	url = https://git.musl-libc.org/cgit/musl
```

Comment:
See my comment here:

- https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1627#issuecomment-2504357536

As far as I can see this change should not be made.